robotintelligence.cpp

```diff
@@ -20,15 +21,22 @@ RobotIntelligence::~RobotIntelligence() {
 
 void RobotIntelligence::run() {
 	initParticles();
+    double lastTime = timeData.getTime();
+    unsigned int lastMeasurementId = laserData.getMeasurementId()-1;

```

Comment:
remove the ID-idea and replace it with a constants.h laserDatafrequence and sync with timeData